daemon/errors.go

```diff
@@ -347,6 +346,15 @@ func errToResponse(err error, snaps []string, fallback errorResponder, format st
 			}
 			handled = false
 		default:
+			// support wrapped errors
+			switch {
```

Comment:
This is the third level of `switch` nesting here :-)

Maybe we should rework existing inner `switch` and replace all the `case *errorType` entries with `case errors.Is()`? Then you'd be able to handle changeconflicterror in the same switch.

Reply:
Yes, that's the intention but, since it requires a lot of changes and it's a bit tangent, I opted for doing the ChangeConflictError only and leaving the rest as a follow-up.